### PR TITLE
Custom highlight color in tabs

### LINKED ISSUE #1543: Use custom highlight colors in tabs
**Describe the solution you'd like**
Currently when a highlight goes off in a chat that you have been pinged in, the taskbar of that chat flashes red. Why not have it so the taskbar of that certain chat tab flashed the color of the highlight which pinged you.  

**Additional context**
If my explanation didn't make enough sense, here is a screenshot of what I am saying. https://imgur.com/a/KKHfq2X This tab would instead flash the color of your chosen highlight

### PR DESCRIPTION
**Pull request by 2qar**
Closes #1543 
Tabs will highlight whatever color is set for that highlight instead of highlighting red.

Example:
Highlight

### CODE REVIEW

**Comment by fourtf** [comment]
what happens if there are multiple highlights?

**Comment by BigHeadGeorge** [comment]
The color of the tab will be whichever highlight happened last. Should there be a priority (eg. username > highlight phrases)?

**Comment by fourtf** [comment]
I'm not sure, perhaps the order of the highlights? Maybe it would also be possible to highlight the tab using multiple colors. None of this is really thought out, just some ideas.

**Comment by leon-richardt** [comment]
> I'm not sure, perhaps the order of the highlights?

That would be nice in combination with #1542. 😄

**Comment by BigHeadGeorge** [comment]
> Maybe it would also be possible to highlight the tab using multiple colors.

Maybe split up like this?
Highlight split

**Comment by fourtf** [comment]
That's what I was thinking off but I'm not sure if it's the right way to go

**Comment by BigHeadGeorge** [comment]
Sorry for the slow response, here's my ideas so far:
* Leaving it and adding #1542 would keep it simple, but won't represent all highlights in that tab
* Splitting the colored part would represent more highlights, but looks kinda ugly and would get cluttered fast
* Flash between the different highlight colors? (probably distracting)

Maybe there could be a setting for this, maybe I'm just overcomplicating it, idk. I think keeping it one color is the right way to go from a visual standpoint.

File: src/widgets/splits/SplitContainer.cpp [hunk 1]
```diff
@@ -200,17 +200,15 @@ void SplitContainer::addSplit(Split *split)
...
                 this->tab_->setHighlightState(state);
             }
-        });
-
-    split->getChannelView().tabHighlightColorRequested.connect(
-        [this](std::shared_ptr<QColor> color) {
             if (color != nullptr)
+            {
                 this->tab_->setHighlightColor(color);
+            }
```
Comment (pajlada) [inline src/widgets/splits/SplitContainer.cpp]: this should jump into the above `if (this->tab_ != nullptr)` scope

**Comment by fourtf** [comment]
I'll consider if this should be added.

**Comment by alazymeme** [comment]
> I'll consider if this should be added.

Have you considered yet?

**Comment by alazymeme** [comment]
...

**Comment by jammehcow** [comment]
@ALazyMeme he's still considering :)

**Comment by xHeaveny** [comment]
time to revive this
image

**Comment by jammehcow** [comment]
@fourtf this is beyond stale now, have you made a decision?

**Comment by MrAuro** [comment]
Is there anyone against this change? This PR has been open for _years_ now. 

Since GitHub issues doesn't have the polls feature that the discussions have, react with 👍 if you are for this change and 👎 if you are against it.

**Comment by Mm2PL** [comment]
@fourtf, @pajlada, @zneix, it's about time to decide if this PR is good or one that we should close.

**Comment by Felanbird** [comment]
We missed 2 years 🎆 🎈 - I'll try to remember next year that Sept 26th is the anniversary

**Comment by Infinitay** [comment]
Do I dare suggest for an option for channels you are subscribed in to be highlighted?

**Comment by fourtf** [comment]
I wasn't aware that this was still open. Looks good to me 👍

**Comment by pajlada** [comment]
Closing in favour of https://github.com/Chatterino/chatterino2/pull/5579

**Pull request closed without merging**